azure-devops/azext_devops/dev/team/user.py

```diff
@@ -13,7 +14,7 @@
 
 
 def get_user_entitlements(top=100, skip=None, organization=None, detect=None):
-    """List users for an organization.
+    """List users for an organization [except for AAD users].
```

Comment:
I think this will show AAD users also if user is directly added in Users tab, but this does not expand AAD group.
So we should tell that, the current text looks incorrect

Reply:
"List users for an organization **[except for users which are added via AAD groups]** "
Does this makes sense @gauravsaralMs  , @geverghe ?